.rubocop.yml

```diff
@@ -9,9 +9,6 @@ AllCops:
         - 'tmp/**/*'
         - 'vendor/bundle/**/*'
 
-Rails:
-    Enabled: true
```

Comment:
why have we disabled this?

Reply:
So the rule was just un-known for rubocop, there is a gem you can install for it but yeah it flags Rails as an un-known

Reply:
huh! Yeah, I must have broken that in a rubocop upgrade at some point over the years, but it only raises it as an error when you run `rubocop`, not `rake`. I think I need to start again with the latest version of rubocop at some point.